js_modules/dagit/packages/core/src/launchpad/LaunchpadRoot.tsx

```diff
@@ -15,11 +17,46 @@ import {
 } from './ConfigEditorConfigPicker';
 import {LaunchpadSessionError} from './LaunchpadSessionError';
 import {LaunchpadSessionLoading} from './LaunchpadSessionLoading';
+import {LaunchpadTransientSessionContainer} from './LaunchpadTransientSessionContainer';
 import {LaunchpadRootQuery, LaunchpadRootQueryVariables} from './types/LaunchpadRootQuery';
 
-const LaunchpadManySessionsContainer = React.lazy(() => import('./LaunchpadManySessionsContainer'));
+const LaunchpadStoredSessionsContainer = React.lazy(() => import('./LaunchpadStoredSessionsContainer'));
 
-export const LaunchpadRoot: React.FC<{repoAddress: RepoAddress}> = (props) => {
+export type LaunchpadType = 'asset' | 'job';
+
+// ########################
+// ##### LAUNCHPAD ROOTS
+// ########################
+
+export const AssetLaunchpad: React.FC<{
+  repoAddress: RepoAddress;
+  sessionPresets?: Partial<IExecutionSession>;
+  assetJobName: string;
+  open: boolean;
+  setOpen: (open: boolean) => void;
+}> = ({repoAddress, sessionPresets, assetJobName, open, setOpen}) => {
+  const title = 'Launchpad (configure assets)';
+
+  return (
+    <Dialog
+      style={{height: '1000px', width: '80%'}}
```

Comment:
I think this may be taller than a 13" MBP screen, could we make it something that is based on the viewport height to be safe (like 90vh?)

Reply:
Good call, done.